src/index.html

```diff
@@ -3,11 +3,351 @@
+        </article>
+
+        <article class="article">
+          <div class="pictures pictures--grid">
+            <div class="pictures__item pictures__item--small">
+              <img
+                class="pictures__image pictures__image--small"
+                src="images/picture-3.jpg"
+                alt="Earphones"
+              >
+            </div>
+
+            <div class="pictures__item pictures__item--small">
+              <img
+                class="pictures__image pictures__image--small"
+                src="images/picture-4.jpg"
+                alt="Earphones"
+              >
+            </div>
+
+          </div>
+
+          <h2 class="title__products-name">Earphones</h2>
+        </article>
+
+        <article class="article">
+          <div class="pictures pictures--grid">
+            <div class="pictures__item pictures__item--small">
+              <img
+                class="pictures__image pictures__image--small"
+                src="images/picture-5.jpg"
+                alt="Smart-Home"
+              >
+            </div>
+
+            <div class="pictures__item pictures__item--small">
+              <img
+                class="pictures__image pictures__image--small"
+                src="images/picture-6.jpg"
+                alt="Smart-Home"
+              >
+            </div>
+          </div>
+
+          <h2 class="title__products-name">Smart Home</h2>
+        </article>
+
+        <div class="section__div-grid">
+          <a href="#" class="button">View all</a>
+        </div>
+
+        <div class="section__article-grid">
+          <article class="article--beige" id="about-us">
+            <div class="pictures__item pictures__item--beige">
+              <img
+                class="pictures__image pictures__image--beige"
+                src="images/picture-7.jpg"
+                alt="Picture"
+              >
+            </div>
+
+            <div class="article--beige__description">
+              <h2 class="description__title">
+                Timeless, for 50 years and counting
+              </h2>
+
+              <p class="description__paragraph">
+                Bang & Olufsen is expanding its Recreated Classics Program with the launch of Beosystems – a limited edition music system that transcends time by bridging the gap between one of our iconic designs from 1972 and today’s cutting-edge digital technology.
+              </p>
+            </div>
+          </article>
+        </div>
+      </section>
+
+      <section class="contact-us" id="contact-us">
+        <h2 class="title title--secondary">
+          Contact us
+        </h2>
+
+        <div class="contact-us__form">
+          <form class="form section__form">
+            <div class="form__item">
+              <label
+                class="form__label"
+                for="name"
+              >  Name
+              </label>
+
+              <input
```

Comment:
Please fix indentations

```suggestion
              <input
                id="name"
                class="form__input"
                type="text"
                placeholder="Name"
                required
              >
```